package.json

```diff
@@ -29,7 +29,7 @@
     "cover": "istanbul cover _mocha && istanbul check-coverage",
     "sass": "./node_modules/node-sass/bin/node-sass ./assets/scss/app.scss ./public/css/app.css --include-path ./node_modules",
     "create:public": "mkdir -p ./public/js ./public/css ./public/images",
-    "hof-transpile": "hof-transpiler ./apps/**/translations/src -w --shared ./node_modules/hof-template-partials/translations",
+    "hof-transpile": "hof-transpiler ./apps/**/translations/src --shared ./node_modules/hof-template-partials/translations/src",
```

Comment:
Indiscriminate copy-pasting without regard for content probably. Most things are.